applications/portal/backend/api/migrations/0010_alter_collaborator_experiment_and_more.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.0.6 on 2022-10-06 15:24
```

Comment:
@afonsobspinto why is this migration needed?

Reply:
Honestly I'm not sure, that appeared from running makemigrations without any changes applied:

Uploading 2023-04-12 09-20-41.mp4…